pkg/capabilities/pb/capabilities_helpers.go

```diff
@@ -149,6 +149,106 @@ func UnmarshalTriggerResponse(raw []byte) (capabilities.TriggerResponse, error)
 	return TriggerResponseFromProto(&tr)
 }
 
+func RegisterToWorkflowRequestToProto(req capabilities.RegisterToWorkflowRequest) *RegisterToWorkflowRequest {
+	config := values.EmptyMap()
+	if req.Config != nil {
+		config = req.Config
+	}
+
+	return &RegisterToWorkflowRequest{
+		Metadata: &RegistrationMetadata{
+			WorkflowId:  req.Metadata.WorkflowID,
+			ReferenceId: req.Metadata.ReferenceID,
+		},
+		Config: values.ProtoMap(config),
+	}
+}
+
+func RegisterToWorkflowRequestFromProto(req *RegisterToWorkflowRequest) (capabilities.RegisterToWorkflowRequest, error) {
+	if req == nil {
+		return capabilities.RegisterToWorkflowRequest{}, errors.New("received nil register to workflow request")
+	}
+
+	if req.Metadata == nil {
+		return capabilities.RegisterToWorkflowRequest{}, errors.New("received nil metadata in register to workflow request")
+	}
+
+	config, err := values.FromMapValueProto(req.Config)
+	if err != nil {
+		return capabilities.RegisterToWorkflowRequest{}, err
+	}
+
+	return capabilities.RegisterToWorkflowRequest{
+		Metadata: capabilities.RegistrationMetadata{
+			WorkflowID:  req.Metadata.WorkflowId,
```

Comment:
why isn't owner populated in any of these calls?

Reply:
honestly, didn't know if it was needed (whereas ReferenceID def is), but I detect a groundswell of support in the comment for adding it ;-), will add it